python/aibrix_kvcache/pyproject.toml

```diff
@@ -0,0 +1,99 @@
+[project]
+name = "aibrix-kvcache"
+version = "0.1.0"
```

Comment:
rest version are aligned with aibrix repo tags. we may consider the version strategy here.